language/changes/6-phantom-type-params.md

```diff
@@ -0,0 +1,103 @@
+# Phantom Type Parameters
+
+- Status: Implemented in Move 1.4
+
+## Introduction
+
+A phantom type parameter is one that doesn't show up at runtime, but is checked statically at
+compile time. Previously, Move's type system didn't make a difference on whether a type parameter
+was phantom, but with this new feature, a struct's type parameter can be explicitly declared as
+phantom. A parameter declared as phantom is not considered when computing the
+[conditional abilities with generic types](3-abilities.md#Conditional-Abilities-with-Generic-Types).
+For this relaxed rule to be sound, Move's type system guarantees that a parameter declared as
+phantom is either not used at all in the struct definition, or it is only used as an argument to
+type parameters also declared as phantom.
+
+## Motivations
+
+A phantom type parameter may come in handy in a couple of situations. For example, the struct
```

Comment:
This first paragraph in the Motivations section seems misplaced. It says nothing about the motivation for phantom type parameters, but instead offers examples from the Diem Framework where they can be used. We should move this elsewhere. Since the other Motivations paragraph refers to these specific examples, I suggest moving this paragraph into the Introduction section.

A few other things:

- Let's also drop the first sentence. This is more substantial than providing something "handy".
- Replace "stdlib" with "the Move standard library"
- The term "notorious" livens up the tone of this document but seems too strong. Maybe "prominent"?